reference/compass.md

```diff
@@ -4,11 +4,11 @@ Derived from [Device](device.md).
 
 ```
 Compass {
-  MFVec3f lookupTable [ ]  # interpolation
-  SFBool  xAxis       TRUE # compute x-axis
-  SFBool  yAxis       TRUE # compute y-axis
-  SFBool  zAxis       TRUE # compute z-axis
-  SFFloat resolution  -1
+  MFVec3f lookupTable [ ]    # [lookup table]
+  SFBool  xAxis       TRUE   # [TRUE, FALSE]
+  SFBool  yAxis       TRUE   # [TRUE, FALSE]
+  SFBool  zAxis       TRUE   # [TRUE, FALSE]
+  SFFloat resolution  -1     # [-1, [0, inf]]
```

Comment:
Woow, that's not obvious to understand... Maybe we should use the standard mathematical representation for sets and intervals, which would translate into: `{ -1, [0, inf) }`. If we use this scientific formalism, which I believe is good, we should also change all the `[TRUE, FALSE]` to `{TRUE, FALSE}` and `[0, inf]` to `[0, inf)`, `[ImageTexture, PROTO]` to `{ImageTexture, PROTO}`, `["symmetric", "active", "passive"]` to `{"symmetric", "active", "passive"}`, etc. We should also remove the square brackets around text describing strings to avoid confusion.

See explanations here: https://en.wikipedia.org/wiki/Set_notation